docs/deprecations.md

```diff
@@ -354,6 +354,18 @@ Adapt to this convention now by setting `provider.apiGateway.shouldStartNameWith
 
 Eventually if you have a strong reason to stick to current convention, you may ensure it's kept after upgrading by setting: `provider.apiName: ${sls:stage}`
 
+<a name="KINESIS_CONSUMER_NAME_CONTAINING_SERVICE"><div>&nbsp;</div></a>
+
+## Kinesis consumer name will be changed which will lead downtime during re-deployment
```

Comment:
Instead of writing of what might happen in the title, let's just stick to something like 

```
## Kinesis consumer naming will be changed to ...
```

and in description mention what might happen during the change (e.g. short downtime), which you already added in the description and it sounds good in my opinion :+1: